controllers/controllers.js

```diff
@@ -30,13 +30,33 @@ exports.createUser = (req, res) => {
     })
     .catch(err => {
       res.status(500).send({
-        message: err.message || "Something broke while creating the product"
+        message: "Username already taken" 
       });
     });
 };
 
 // Add an exercise to a given user
 exports.addExercise = (req, res) => {
+  const userId = req.body.userId;
+  const description = req.body.description;
+  const duration = req.body.duration;
+  const date = req.body.date || null; // Not sure if this is right bc date defaults to Date.now in user.model.js
```

Comment:
|| null does nothing
I think it's fine to rely on defaults in the data model but be careful not to use an undefined object in this route